src/js/components/history/HistoryTable.jsx

```diff
@@ -6,27 +6,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import * as SubmissionListHelper from '../../helpers/submissionListHelper';
-import * as UtilHelper from '../../helpers/util';
-import * as Icons from '../SharedComponents/icons/Icons';
+import * as SubmissionListHelper from 'helpers/submissionListHelper';
+import * as UtilHelper from 'helpers/util';
+import * as Icons from 'components/SharedComponents/icons/Icons';
 
 const propTypes = {
-    submissionID: PropTypes.string
-};
-
-const defaultProps = {
-    submissionID: ''
+    submissionID: PropTypes.string.isRequired
 };
 
 export default class HistoryTable extends React.Component {
     constructor(props) {
         super(props);
 
-        this.isUnmounted = true;
```

Comment:
It's unnecessary, since once the component instance is unmounted it will not be rendered again. `componentWillUnmount` should generally be used to do things like cancel API requests, stop timers, etc.